build/cmake/find/FindQwt.cmake

```diff
@@ -30,46 +30,46 @@
 
 if(UNIX)
 
-  find_path(QWT_INCLUDE_DIR qwt.h
-            PATHS /usr
-                  /usr/local
-                  /usr/local/qwt
-            PATH_SUFFIXES include
-                          qwt
-                          include/qwt
-                          lib/qwt.framework/Headers)
+	find_path(QWT_INCLUDE_DIR qwt.h
+			  PATHS /usr
```

Comment:
Note the space before PATHS. It should not exist. Also, use only one additional tab, it seems that you are using two here.